pallets/xcmp-handler/src/lib.rs

```diff
@@ -663,6 +664,9 @@ pub trait XcmFlowSelector {
 
 impl XcmFlowSelector for () {
 	fn flow_for(_para_id: ParaId) -> XcmFlow {
+		if _para_id == ParaId::from(2000) {
+			return XcmFlow::Alternate;
+		}
```

Comment:
This flow selector is required for Shiden Dev(Shibuya). Maybe we should update our zombienet script to make the paraId the same as Rocstar’s, to save some chain code here?

Reply:
Based on our earlier conversation, I'll update the xcmp-handler pallet to accept dynamic values through genesis and governance for the flow selection.

Reply:
Okay, sounds good!